app/models/destination.rb

```diff
@@ -1,5 +1,12 @@
 class Destination < ApplicationRecord
     validates :city, presence: true, uniqueness: true
     validates :country, presence: true
-    validates :currency, presence: true, length: { maximum: 3, message: "doesn't have a valid size" } 
+    validates :currency, presence: true, length: { maximum: 3, message: "doesn't have a valid size" }
+    
+    def self.get_weather(city)
```

Comment:
Way did you add this in model?
Please find a place to move this.

Reply:
The logic was added in the model solely for testing the API, this will be moved to another place in the next commits.